src/integ_test_resources/common/scripts/device_config_builder.py

```diff
@@ -26,17 +26,18 @@ class DeviceConfigBuilder:
     """
 
     def __init__(self, platform: str):
-        if not platform in ['android', 'ios']:
-            raise Exception('Platform must be one of: android, ios.')
+        if platform not in ["android", "ios"]:
```

Comment:
Perhaps extract to a constant?

```python
PLATFORMS = ('ios', 'android')

if platform not in PLATFORMS:
    raise Exception(f"Platform must be one of {', '.join(PLATFORMS)}.")
```

Reply:
I did you one better, and used the `Platform` enum :)